src/main/java/uk/gov/hmcts/reform/sscs/reference/data/mappings/CancellationReason.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.sscs.reference.data.mappings;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum CancellationReason {
+    WITHDRAWN("withdrawn", "Withdrawn", "", "BBA3"),
+    STRUCK_OUT("struckOut", "Struck Out", "", "BBA3"),
+    PARTY_UNABLE_TO_ATTEND("partyUnableToAttend", "Party unable to attend", "", "BBA3"),
+    EXCLUSION("exclusion", "Exclusion", "", "BBA3"),
+    INCOMPLETE_TRIBUNAL("incompleteTribunal", "Incomplete Tribunal", "", "BBA3"),
+    LISTED_IN_ERROR("listedInError", "Listed in error", "", "BBA3"),
+    OTHER("other", "Other", "", "BBA3"),
+    NO_LONGER_READY_FOR_HEARING("noLongerReadyForHearing", "No longer ready for hearing", "", ""),
+    SETTLED("settled", "Settled", "", ""),
+    JUDICIAL_DIRECTION("judicialDirection", "Judicial direction", "", ""),
+    FEE_NOT_PAID("feeNotPaid", "Fee not paid", "", ""),
+    PARTY_DID_NOT_ATTEND("partyDidNotAttend", "Party did not attend", "", ""),
+    LAPSED("lapsed", "Lapsed", "", "");
+
+    private final String key;
+    private final String value;
+    private final String description;
+    private final String serviceCode;
```

Comment:
The document provided does not contain BBA3 for those fields, I can add them if require me updated me.

Reply:
Its less add it in, more do we need them ? e.g. FEE NOT PAID, we dont have fees in SSCS. Trouble is, you try to use any of those others, I imagine they will fail as they are not using our service code.

Reply:
Should probably hardcode this as BBA3 then?

Reply:
Removed

Reply:
It's provided on the documents so I think its worth while keeping as I assume the business have some type of plan for it at some point.

Reply:
I think we should either remove these altogether and let the callers set the service code if needs be (it's a constant in hearings api). Do we know if the presence of our service code denotes whether we use a certain field?

Worried that if we need to change our service code we're making more work for ourselves.
@wmarshid @ss-evoco @MohammedHMCTS

Reply:
I don't think its worth the time to go back though and do this, We can raise another PR as its not efficient to block over minor inconvenience.

Reply:
Agreed with @MohammedHMCTS stop trying to predict the future Del you don't have a crystal ball!

Reply:
I was only able to skim read the above but I think we are asking 'Do we need to add the service code to every object?'. The service code is our own and will always be the same. I think it would be best to just have one constant with our service code in it and then whenever we need to send it, we pass the constant. Then if we ever need to change it, we change it in the one place and everything auto works. That sort out this thread ?

Reply:
Nothing actually uses the serviceCode from these, are you not the one trying to predict the future?

Reply:
Going to waste time going back though and changing them but like I am not project lead so if you got time to wait I can sort it Monday or you can reassign the ticket to someone else as it was not specified in the ticket to not include information https://tools.hmcts.net/jira/browse/SSCS-10273 but instead include all the information from the documents provided.

Reply:
This ticket has been in progress for over 3 weeks ranked at 5 story points, When it has taken 16 days as of up to now.